lib/__tests__/ietf.spec.ts

```diff
@@ -31,9 +31,9 @@
 
 import {CookieJar, parseDate} from '../cookie'
 import url from 'url'
-import parserData from '../../test/ietf_data/parser.json'
-import bsdExampleDates from '../../test/ietf_data/dates/bsd-examples.json'
-import exampleDates from '../../test/ietf_data/dates/examples.json'
+import parserData from './data/parser'
+import bsdExampleDates from './data/dates/bsd-examples'
+import exampleDates from './data/dates/examples'
```

Comment:
Changes to our tsconfig mean it's easier to have the test data within the `./lib` directory, and it's easier if they're TS files, rather than JSON.